### ISSUE TITLE
Prototype solutions for tag component

### ISSUE THREAD

**Issue by hazalarpalikli** [alphagov/govuk-frontend#6344]
## What
This ticket needs to be done after we have done the investigation into understanding the issues with the tag component.  Once we are clear on the issues with the tag component, we need to sketch out solutions to solve them. 

Related documents:

- https://docs.google.com/document/d/1lpAZ07DcKqktidQ-gOpETJXTjAYNKmEPqQRI-8d7Bdc/edit?tab=t.53od11rhya90#heading=h.35hfv1u2d200
- https://github.com/alphagov/govuk-frontend/issues/6267#issuecomment-3370994667

Possible solutions @CharlotteDowns suggested:

- Give tags an outline of 1px border
- Would adding borders make them look more like clickable things, and if so, do we allow for that?
- When thinking about their purpose, is it a tag or is it a label?
- Do we need to have a deeper investigation into tags in more detail, outside of the colour change? TL:DR Just working on the colour change seems ok for now.

## Why
So that we have an accessible, working component.

## Who needs to work on this
Designer

## Who needs to review this
The whole team 

## Done when

- [x] Sketch out solutions 
- [x] Agree on a solution
- [x] Prototype the solution in the review app

**Comment by hazalarpalikli** [comment]
Decision: Following a review with @owenatgov and @mia-allers-gds, this ticket can be closed.

## Changes we are making to the tag component

### Border & Padding
- Border: Added 1px border with a 1px border-radius to improve text legibility.
- Padding: Increased top and bottom padding to 4px for better visual balance. 

### Border colour
Adding the following border colour to the tags.

- Default : Blue shade 50%  `#0F385C`
- Grey : Black tint 25% `#484949`
- Purple: Purple shade 50% `#2A1950`
- Turquoise: Teal shade 50% `#116568`
- Light blue: Blue shade 50% `#0F385C`
- Yellow: Yellow shade 50% `#806F00`
- Orange: Orange shade 50% `#7A3C1C`
- Red: Red shade 50% `#7A3C1C`
- Pink :  Magenta shade 50% `#651B3E`
- Green :  Green shade 50% `#09442D `

### Next steps
- Task List Page Layout: A follow-up is required to address a layout issue on the Task List page. The increased padding on the tag component has created a visual imbalance within table cells.

- Proposed Solution: Investigate applying a negative margin to the tag component to correct the alignment, as suggested by @owenatgov 

Designs could be seen in this pull request:  https://github.com/alphagov/govuk-frontend/pull/6340
